src/transformers/tokenization_transfo_xl.py

```diff
@@ -280,6 +293,108 @@ def _tokenize(self, line, add_eos=False, add_double_eos=False):
             return symbols
 
 
+class _TransfoXLDelimiterLookupTokenizer(BaseTokenizer):
```

Comment:
Can we move this class upstream in `tokenizers` now that we have a word-level model?

It would be more consistent.

Reply:
cc @n1t0 what do you think ? I can put the content of this into tokenizers.implementations

Reply:
cf comment for `OpenAIGPTCharBPETokenizer`